### ISSUE TITLE
Docker on Windows

### ISSUE THREAD

**Issue by mmh352** [innovationOUtside/Open_Computing_Lab_Guide#1]
This is more of a note for future updates. With the latest 10 May 2020 Windows 10 update, Docker Desktop installs without any hacks required. The user has to manually download the Windows Subsystem for Linux 2, but they are guided to that by Docker as well.

**Comment by psychemedia** [comment]
@mmh352 Ah, thanks.. Does that depend on what sort of Windows 10 you are using (I thought only some flavours supported, I dunno, stuff. (Home, Professional etc etc all seem to support different things? But it's been years since I last used a windows m/c.))

Just by the by, chunks of the content of this repo will disappear at some point, or at least, will be presented via a Github submodule pulling from https://github.com/OpenComputingLab/userguide. More discussion here :https://github.com/innovationOUtside/Open_Computing_Lab_Guide/issues/2  (and yes, I'm already getting twitchy about how the levels of indirection might explode!)

**Comment by mmh352** [comment]
This is for Windows Home. As far as I understand Windows Pro already had the necessary tech to run Docker without any hacks, so nothing has changed there. I don't really run Windows myself, but I just finished building the first TT284 docker image, so I wanted to test how well that would work on Windows.

**Comment by psychemedia** [comment]
Ah, I missed that...

WSL2 will be generally available in Windows 10, version 2004

https://devblogs.microsoft.com/commandline/wsl2-will-be-generally-available-in-windows-10-version-2004/

So is that Windows 10 generally?

And https://www.docker.com/blog/docker-desktop-for-windows-home-is-here/

> We are now pleased to announce that we have completed the work to enable experimental support for Windows Home WSL 2 integration. This means that Windows Insider users on 19040 or higher can now install and use Docker Desktop!

**Comment by mmh352** [comment]
Yes, that is Windows 10 generally. I have Windows 10 Home on my laptop and docker installed fine on that after upgrading Windows 10 to the May 2020 release (which is version 2004). I assume that if it runs on Home, it will run on Pro too :-).
